### PR TITLE
[chore] migrate ci to gha

### PR DESCRIPTION
**Pull request by aricart**

### CODE REVIEW

File: v2/go.mod [hunk 1]
```diff
@@ -2,9 +2,11 @@ module github.com/nats-io/jwt/v2
 
 go 1.18
 
-require github.com/nats-io/nkeys v0.3.1-0.20221205184623-5d8a6730c42c
+require (
+	github.com/nats-io/nkeys v0.3.1-0.20221205184623-5d8a6730c42c
```
Comment (wallyqs) [inline v2/go.mod:6]: something unreleased from nkeys is needed?

Comment (aricart) [reply v2/go.mod:6]: possibly that is xkeys - let me check what that is

Comment (wallyqs) [reply v2/go.mod:6]: should we make an nkeys release? 
https://github.com/nats-io/nkeys/compare/v0.3.0...main

**Review by wallyqs — APPROVED** [review]
LGTM

File: v2/go.mod (hunk 1 above)
Comment (aricart) [reply v2/go.mod:6]: yes this for the unreleased xkeys - we will need to release that when we release the lib

**Pull request merged**